tests/foreman/api/test_docker.py

```diff
@@ -828,12 +755,11 @@ def test_promote_multiple_docker_repo_content_view(self):
         cvv = content_view.read().version[0]
         self.assertEqual(len(cvv.read().environment), 1)
 
-        for i in range(1, randint(2, 5)):
```

Comment:
Please, return as it was. I put `for i in range(1, randint(2, 5)):` intentionally as `i+2` is not logical. Of course, it can be better to have `for i in range(2, randint(3, 6))`